### DISCUSSION TITLE
Please return old splash screen with the beaver itself

### DISCUSSION THREAD

**Discussion by mcekovic** [dbeaver/dbeaver#13274] [Ideas]
Please return the old splash screen with a beaver itself, it was extraordinary and one of the reasons I started using DBeaver. Please return it at least for the Community version!
New splash screen is pretty plain and a bit 'enterprisey'. I understand it is to drive DBeaver adoption in the enterprises, but in order to be adopted in the enterprises, it needs to be adopted in the community first, and old splash screen was perfect for the purpose!

**Comment by tkingsbury42** [comment]
Amazing. I was here to complain about the exact same thing, but mcekovic beat me to it by just a few hours.
Your Beaver images are absolutely brilliant. Meanwhile, the splash screen with the enterprise-grade red circles is terribly sad and depressing.

**Comment by tkingsbury42** [comment]
Nice!

![image](https://user-images.githubusercontent.com/36487732/157727335-8286778b-7d8e-432f-a760-87276ba4b181.png)
